docs/dev/static_libaries.md

```diff
@@ -0,0 +1,152 @@
+# Building OpenVINO static libraries
+
+## Contents
+
+- [Introduction](#introduction)
+- [Configure OpenVINO runtime in CMake stage](#configure-openvino-runtime-in-cmake-stage)
+- [Build static OpenVINO libraries](#build-static-openvino-libraries)
+- [Link static OpenVINO runtime](#link-static-openvino-runtime)
+- [Static OpenVINO libraries + Conditional compilation for particular models](#static-openvino-libraries--conditional-compilation-for-particular-models)
+- [Building with static MSVC Runtime](#building-with-static-msvc-runtime)
+- [Limitations](#limitations)
+- [Additional Resources](#additional-resources)
+
+## Introduction
```

Comment:
I propose to add `System requirements` at the beginning of this page.

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/16063/commits/562c6c32912eb936d9ca384c7caa9cf772ce2fa2